src/components/AccountSelector.tsx

```diff
@@ -30,7 +30,7 @@ const Main = () => {
       <label style={{ fontSize: '12px' }}>
         <SelectedAccountDiv className="text-start mb-1">{activeAccount}</SelectedAccountDiv>
         <LevelStatusDiv>
-          <label style={{ paddingRight: '20px' }}>JOURNEY: {level.toUpperCase()}</label>
+          <label className="pr-3">JOURNEY: {level.toUpperCase()}</label>
```

Comment:
I don't know why, but `pr` is not working for the text, padding down works, but padding-right doesn't, that's why I use style instead of bootstrap :disappointed:

Reply:
Argh, they changed the api. It is now `pe-3` (_padding end 3_) :P